docs/OV_Runtime_UG/supported_plugins/GNA.md

```diff
@@ -10,9 +10,7 @@ to save power and free CPU resources.
 
 The GNA plugin provides a way to run inference on Intel® GNA, as well as in the software execution mode on CPU.
 
-## Devices with Intel® GNA
```

Comment:
looks like we can remove `## Introducing the GNA Plugin` above and start with no title..